docs/modules/ROOT/pages/how-tos/aks_recover_from_backup.adoc

```diff
@@ -0,0 +1,148 @@
+= Recover cluster objects from Azure Blob Storage backup
+
+== Prerequisites
+
+* Executables used in this guide:
+** `kubectl`
+** https://github.com/kislyuk/yq[yq: Command-line YAML/XML/TOML processor] installable via `pip3 install yq`
```

Comment:
This is a different `yq` than the one we generally use (https://github.com/mikefarah/yq), and seems to have nothing in common with the one by mikefarah which our documentation commonly refers to.

I think we should keep our documentation referring to only one `yq` implementation. Since there's already multiple documentations that refer to github.com/mikefarah/yq, I'd suggest going with that one, and requiring v4 or newer.

I've added comments below to update the commands to work with that yq.

Reply:
Applied.